content/projects/github-api-consume/_index.md

```diff
@@ -72,6 +72,6 @@ get_pull_requests("Umuzi-org", "ACN-syllabus", "2022-03-01", "2022-03-10")
 
 ### Instructions for reviewers
 - Ensure that the function works with any GitHub User and repository name, meaning that the function should not only work with one User/repo name. 
-- Ensure that the function prints for JavaScript and python, the function should return the output.
+- Ensure that for JavaScript the function should print the output, and for python the function should return the output
```

Comment:
would prints not work better than should print in this context, the sentence feels off.

Also, returns would probably work better than should return?